q2_dada2/tests/data/single_sample.tsv

```diff
@@ -0,0 +1,2 @@
+#OTU ID	L1S208	L1S257	L1S57	L1S76	L2S155	L2S175	L2S309	L2S357	L3S294	L3S313	L4S112	L4S63	L5S155	L5S174	L6S20	L6S68
```

Comment:
Are we testing a single sample (`single_sample.tsv`) or a single feature (`single_feature.tsv`)? If its the first can you explain in more detail? Thanks!

Reply:
good catch, that should be `single_feature` I'll rename this file.

Reply:
Please rename the tests etc as [necessary](https://github.com/qiime2/q2-dada2/pull/127#pullrequestreview-350268631), too!